src/main/io/osd.c

```diff
@@ -32,7 +32,7 @@
 #include "platform.h"
 
 FILE_COMPILE_FOR_SPEED
-
+#define USE_OSD
```

Comment:
OSD should not define `USE_OSD`

Reply:
Apologies for that - a temporary change that should not have been released.
I have removed it.